src/components/ui/faqitem.tsx

```diff
@@ -0,0 +1,25 @@
+import {
+  AccordionContent,
+  AccordionItem,
+  AccordionTrigger
+} from "@/components/ui/accordion"
+import * as React from "react"
+
+export function FAQItem({
+  children,
+  title
+}: {
+  children: React.ReactNode
+  title: string
+}) {
+  return (
+    <AccordionItem value={title} className="border-none">
+      <AccordionTrigger className="mb-0 w-full rounded px-2 py-4 text-left font-normal transition hover:bg-slate-700 hover:no-underline">
+        <h3 className="text-xl">{title}</h3>
+      </AccordionTrigger>
+      <AccordionContent className="mt-0 p-2 pt-0 text-base text-stone-400">
+        {children}
+      </AccordionContent>
+    </AccordionItem>
+  )
+}
```

Comment:
src/components/ui/* is for highly reusable components, by nature a faq component isn't very reusable. It should be placed in app/student/recruitment/components/faqItem.tsx

Reply:
The reason I placed it there is because it is also re used on the packages page. If I move this code to the student specific folder, I would have to create duplicate code in the company folder too.

Reply:
I see, I would probably create a new folder in that case, for example /components/shared. The reason I'm very picky about this is that if we do not make this distinction this folder will grow and become bloated quite quickly. The component should preferably be self contained too which isn't the case for this component since it relies on the <Accordion> component wrapping it